src/from_geo.rs

```diff
@@ -62,8 +62,20 @@ impl<'a> TryFrom<&'a LineString<f64>> for GGeom<'a> {
 struct LineRing<'a>(&'a LineString<f64>);
 
 /// Convert a geo_types::LineString to a geos LinearRing
-/// a LinearRing should be closed so close the geometry if needed
-/// What happens if the linestring is empty?
+/// Empty LineRing are valid
+/// The rules for validation and construction are those followed by
+/// [Shapely](https://shapely.readthedocs.io/en/latest/manual.html#linearrings)
+/// If the input LineString is not closed, then the resulting LineRing closes the construction
```

Comment:
This comment is better! 

But keep in mind, unless we change the behavior, it's potentially misleading - we don't close only the unclosed rings - we also "close" linestrings of 3 points, regardless of if their closed or unclosed.